prow/oss/cluster/hook-ghapp.yaml

```diff
@@ -28,6 +28,7 @@ spec:
         image: gcr.io/k8s-prow/hook:v20211008-562307626f
         imagePullPolicy: Always
         args:
+        - --webhook-path=/ghapp-hook/hook
```

Comment:
nit: The `/hook` suffix is no longer needed. We could just use `/ghapp-hook` here and in the ingress rule.

Reply:
done